larray/viewer/api.py

```diff
@@ -0,0 +1,301 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import sys
+import traceback
+from collections import OrderedDict
+import numpy as np
+import larray as la
+
+from qtpy.QtWidgets import (QApplication)
+from larray.viewer.view import (MappingEditor, ArrayEditor, SessionComparator, ArrayComparator)
```

Comment:
the parentheses are not necessary either but that's ok if you prefer to have them